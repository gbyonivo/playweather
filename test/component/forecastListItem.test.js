import React from 'react';
import renderer from 'react-test-renderer';
import ForecastListItem from '../../src/components/forecastListItem';

describe('ForecastListItem component', () => {
  it('renders correctly', () => {
    const actual = renderer.create(
      <ForecastListItem
        item={{ main: { temp_min: '', temp_max: '', humidity: '' }, weather: [], dt_txt: 'dttxt' }}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});
