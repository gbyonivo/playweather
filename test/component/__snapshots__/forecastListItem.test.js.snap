// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastListItem component renders correctly 1`] = `
<div
  className="forecastListItem"
>
  <h4>
    dttxt
  </h4>
  <div
    className="weatherIcons"
  >
    <div
      className="weatherIcons"
    />
  </div>
  <div>
    temp-min: 
    
  </div>
  <div>
    temp-max: 
    
  </div>
  <div>
    humidity: 
    
  </div>
</div>
`;
